packages/service-integration/scripts/ooil.bash

```diff
@@ -6,8 +6,7 @@ set -o nounset
 set -o pipefail
 IFS=$'\n\t'
 
-
-IMAGE_NAME="${DOCKER_REGISTRY:-itisfoundation}/service-integration:${DOCKER_IMAGE_TAG:-master-github-latest}"
+IMAGE_NAME="${DOCKER_REGISTRY:-itisfoundation}/service-integration:${OOIL_IMAGE_TAG:-master-github-latest}"
```

Comment:
line 14 below, use long option names. Also I am not sure you need ```-it``` when you run headless